src/javatests/com/google/cloud/bigquery/dwhassessment/extractiontool/dumper/DataEntityManagerTempTestImpl.java

```diff
@@ -0,0 +1,52 @@
+/*
```

Comment:
This file seems really similar to DataEntityManagerTesting.java. How about creating a testing class, that can use either String or OutputStream under the hood?

Also, I believe it should be called Fake explicitly.
Consider moving it under `testing` directory to distinguish Fakes and Test suites

Reply:
Merged and renamed. Will move it, and other testing helper/utils to a `testing` directory in another CL.